recipes/sklearn-contrib-lightning/run_test.py

```diff
@@ -0,0 +1,8 @@
+import lightning
+import nose
+import os
+
+lightning_dir = os.path.dirname(
+    os.path.abspath(lightning.__file__))
+os.chdir(lightning_dir)
+nose.run(module=lightning)
```

Comment:
It passed smoothly. So let's leave it there. I would remove only if the test get heavy and the CIs start to choke.